grease/src/Grease/GitRev.hs

```diff
@@ -1,21 +1,29 @@
-{-|
-Copyright        : (c) Galois, Inc. 2024
-Maintainer       : GREASE Maintainers <grease@galois.com>
--}
-
 {-# LANGUAGE TemplateHaskell #-}
 
--- | These are placed in their own module to minimize the cost of recompilation
+-- \| These are placed in their own module to minimize the cost of recompilation
 -- due to Template Haskell.
+
+-- |
+-- Copyright        : (c) Galois, Inc. 2024
+-- Maintainer       : GREASE Maintainers <grease@galois.com>
```

Comment:
The formatting here is a bit weird. How about:

```suggestion
-- |
-- Copyright        : (c) Galois, Inc. 2024
-- Maintainer       : GREASE Maintainers <grease@galois.com>
--
-- These are placed in their own module to minimize the cost of recompilation
-- due to Template Haskell.
```

Reply:
Looks like I accidentally touch comments. I revert them to original state.